### DISCUSSION TITLE
Recovering a tour in MTB

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1072] [General]
Hello
How can I retrieve an old tour in MTB database similar to the one I performed to day? (a few years ago)

**Comment by Thompson-ongithub** [comment]
Hi Gringojack
It might be a stupid question, but have you tried the search function (by pressing <crtl>+K)?
https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/show-tours/search

**Comment by wolfgang-ch** [comment]
* [Compare Tours by Altitude ](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tool-compare-tours)
* [Compare Tours by Geo Location](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tool-compare-tours-by-geo)

**Reply by gringojack** [reply]
Hi
Thank you
Yes I tried all of these pieces of info
That works fine
